### DISCUSSION TITLE
Station Offline But Still Streaming

### DISCUSSION THREAD

**Discussion by dekakast** [AzuraCast/AzuraCast#6948] [Q&A] [answered]
I tried reloading the station but it still shows offline. Music is still streaming. I then restarted broadcasting, but still offline. I am running 19.4 stable. Icecast and Liquidsoap both show they are running. My CPU load is only 8% and I have plenty of Memory. I flushed the local cache on the server. I looked at the logs but not sure what I should be focusing on. Appreciate any suggestions on how to troubleshoot.

**Comment by Heimatradio** [comment]
same here with Icecast, but only with FLAC- and OGG streams, Icecast with MP3- and AAC-Streams working
https://github.com/AzuraCast/AzuraCast/discussions/6386

**Comment by dekakast** [comment]
I am guessing it has something to do with toggling on "Use High-Performance Now Playing Updates" in System Settings.

**Comment by dekakast** [comment, marked as answer]
I toggled off "Use High-Performance Now Playing Updates" in System Settings and the stations came back online. The downside is now I cannot use SSE.
